feature(Set Is Medical from Employee Jobtitle)

lomamech opened the pull request:
- Remove property is_medical from Employee
- Add property is_medical in table title_employee
- Update modules 

Closes #7170.

lomamech committed:
Improve key of translation, Use Job Function instead function and fix placeholder message

jmcameron reviewed (commented):
The code changes look okay.

I tested it with bhima_test and would like two fixes:
 - In the employee registry when I try to do a search by "Job Title", it does not work.
 - Please update the employee registry to it remembers the column selections.  Currently it forgets the column selections every time you leave the page.

I did not test this by doing a payroll. I encourage you to verify that the payroll process works correctly with this change.

lomamech committed:
bug(Fix Search employee by job Title)
- Fix error and bug on component Employee Job title select

lomamech committed:
feature(Set Is Medical from Employee Jobtitle)
- Remove property is_medical from Employee
- Add property is_medical in table title_employee
- Update modules
closes #7170

jmcameron reviewed (approved):
Chris, thanks for your updates.  I tested them and they work okay.
